Add sectionsharingan aa1 to master

petschna opened the pull request:
**Checklist:**

 <!--- Make sure your PR is documented and tested before submission. Put an `x` in all the boxes that apply: -->
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?


**Type of Changes:**

 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)


**What is the current behavior? (link to any open issues here)**
- Currently the SectionSharingan is outdated and does not present the Vaporwave style. #24 


**What is the new behavior (if this is a feature change)?**
- Now there is a SectionSharingan which shows the Vaporwave style and fits the rest of the website with the overhauled design. Closes #24

petschna committed:
Add SectionSharingan organism

The SectionSharingan organism has
been added to the environment.

petschna committed:
Add SectionSharingan to index.js

The SectionSharingan has
been added to the index.js.

petschna committed:
Add SectionSharingan to PageRoot

The SectionSharingan has
been added to the PageRoot.js.

petschna committed:
Add assets to sharingan

The assets of the sharingan section
have been added to the environment.

petschna committed:
Add team assets to sharingan

The team assets of the sharingan section
have been added to the environment.

petschna committed:
Add sculpture assets to sharingan

The sculpture assets of the sharingan section
have been added to the environment.

petschna committed:
Rework sharingan background colors

The background colors of the different sharingan
sections have been modified to fit the vaporwave style.

petschna committed:
Rework mitsudomoe colors

The colors of the mitsudomoe windows have been
changed to fit the vaporwave style.

petschna committed:
Add sculpture overlays

The sculpture overlays have been added
to the environment to fit in the sculpture images.

petschna committed:
Rework sculpture and overlay naming

The naming of the sculptures and the overlay have been
changed to be more clear and precise to the reader.

petschna reviewed (commented):
Should be ok to merge.

petute reviewed (approved):
Should be ok to merge.

kleberbaum commented on src/components/organisms/SectionSharingan/index.jsx:
```diff
@@ -0,0 +1,280 @@
+                <div class="sculptures">
+                  <img class="sculpture-overlay1" src={sculpture1}></img>
+                  <img class="sculpture-overlay2" src={sculpture2}></img>
+                </div>
```
```suggestion
                <div class="sculpture-overlay">
                  <img class="sculpture-img1" src={sculpture1}></img>
                  <img class="sculpture-img2" src={sculpture2}></img>
                </div>
```

kleberbaum commented on src/components/organisms/SectionSharingan/index.jsx:
```diff
@@ -0,0 +1,280 @@
+                    <img src={Mitsudomoe2IMG} alt="mitsudomoe2" />
+                  </MDBBox>
+                </MDBBox>
+                <div class="sculptures">
+                  <img class="sculpture-overlay1" src={sculpture1}></img>
+                  <img class="sculpture-overlay2" src={sculpture2}></img>
+                </div>
```
```suggestion
                <div class="sculptures-overlay">
                  <img class="sculpture-img1" src={sculpture1}></img>
                  <img class="sculpture-img2" src={sculpture2}></img>
                </div>
```

kleberbaum commented on src/components/organisms/SectionSharingan/index.jsx:
```diff
@@ -0,0 +1,280 @@
+                    <img src={Mitsudomoe3IMG} alt="mitsudomoe3" />
+                  </MDBBox>
+                </MDBBox>
+                <div class="sculptures">
+                  <img class="sculpture-overlay1" src={sculpture1}></img>
+                  <img class="sculpture-overlay2" src={sculpture2}></img>
+                </div>
```
```suggestion
                <div class="sculptures-overlay">
                  <img class="sculpture-img1" src={sculpture1}></img>
                  <img class="sculpture-img2" src={sculpture2}></img>
                </div>
```

kleberbaum commented on src/components/organisms/SectionSharingan/sharingan.scss:
```diff
@@ -0,0 +1,459 @@
+  display: inline;
+}
+
+.sculpture-overlay1 {
+  position: absolute;
+  width: 400px;
+  margin-left: 1100px;
+  margin-top: -150px;
+}
```
```suggestion
.sculpture-img1 {
  position: absolute;
  width: 400px;
  margin-left: 1100px;
  margin-top: -150px;
}
```

kleberbaum commented on src/components/organisms/SectionSharingan/sharingan.scss:
```diff
@@ -0,0 +1,459 @@
+  margin-top: -150px;
+}
+
+.sculpture-overlay2 {
+  position: absolute;
+  width: 400px;
+  margin-right: 400px;
+  margin-left: -360px;
+  margin-top: -450px;
+  transform: scaleX(-1);
+}
```
```suggestion
.sculpture-img2 {
  position: absolute;
  width: 400px;
  margin-right: 400px;
  margin-left: -360px;
  margin-top: -450px;
  transform: scaleX(-1);
}

```

kleberbaum commented on src/components/organisms/SectionSharingan/sharingan.scss:
```diff
@@ -0,0 +1,459 @@
+  }
+}
+
+
+.sculptures {
+  position: relative;
+  display: inline;
+}
```
```suggestion
.sculptures-overlay {
  position: relative;
  display: inline;
}
```

kleberbaum reviewed (changes requested):
Should be ok to merge after a few changes.

petschna committed:
Apply suggestions from code review

Reworked sculptures naming

The naming of the sculpture classes were 
changed to be more understanding.

Co-authored-by: Florian Kleber <kleberbaum@erebos.xyz>

petschna reviewed (commented):
Should be ok to merge.

kleberbaum reviewed (approved):
Should be ok to merge.

kleberbaum committed:
Merge branch 'master' into add-sectionsharingan-aa1
